templates/traefik-forward-auth.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: traefik-forward-auth
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: https://github.com/jongiddy/traefik-forward-auth-chart/archive/oidc-issuer.tar.gz
+    version: 0.1.1
+    values: |
+      ---
+      replicaCount: 1
+
+      image:
+        repository: jongiddy/traefik-forward-auth
```

Comment:
when can we get it into a `mesosphere/traefik-forward-auth` repo or into `mesosphere/charts` 🙏  ?

Reply:
This will take a little time. It's not just creating the GitHub repo. I also need to get it uploaded to Mesosphere's Docker repo.

Reply:
I think it's OK to for now while we're working on porting it to `mesosphere/traefik-forward-aut`

Reply:
@jongiddy sounds good 👍, but please let's add a jira ticket and a comment to do it in the future.

Reply:
https://jira.mesosphere.com/browse/DCOS-56251